lib/jcsda-emc/spack-stack/stack/stack_env.py

```diff
@@ -108,11 +109,39 @@ def env_dir(self):
     def add_includes(self, includes):
         self.includes.extend(includes)
 
+    def get_lmod_or_tcl(self, site_configs_dir):
+        site_modules_yaml_path = os.path.join(site_configs_dir, "modules.yaml")
+        with open(site_modules_yaml_path, "r") as f:
+            site_modules_yaml = syaml.load_config(f)
+        lmod_or_tcl_list = site_modules_yaml["modules"]["default"]["enable"]
+        assert (
+            lmod_or_tcl_list
+        ), "Set 'modules:default:enable' in site modules.yaml, or use --modulesys {tcl,lmod}"
+        if len(set(lmod_or_tcl_list)) > 1:
+            logging.warning(
+                "WARNING: Multiple lmod/tcl settings found for %s; using the first"
```

Comment:
I think this should be an error that leads to an abort.

Reply:
Okay sounds good